lib/l10n/app_it.arb

```diff
@@ -0,0 +1,260 @@
+  "play_next": "Riproduci seccessivo",
+  "removed_track_from_queue": "Rimossa {track} dalla coda",
+  "remove_from_queue": "Rimuovi dalla coda",
+  "remove_from_favorites": "Rimuovi dai preferiti",
+  "save_as_favorite": "Salva come preferito",
+  "add_to_playlist": "Aggiungi alla playlist",
+  "remove_from_playlist": "Rimuovi dalla playlist",
+  "add_to_blacklist": "Aggiungi alla blacklist",
+  "remove_from_blacklist": "Rimuovi dalla blacklist",
+  "share": "Condividi",
+  "mini_player": "Mini Player",
+  "slide_to_seek": "Slide to seek forward or backward",
+  "shuffle_playlist": "Shuffle playlist",
+  "unshuffle_playlist": "Unshuffle playlist",
+  "previous_track": "Traccia Precedente",
+  "next_track": "Traccia Successiva",
+  "pause_playback": "Pausa Playback",
+  "resume_playback": "Riprendi Playback",
+  "loop_track": "Brano in Loop",
+  "repeat_playlist": "Ripeti playlist",
+  "queue": "Coda",
+  "alternative_track_sources": "Sorgenti dei brani Alternative",
+  "download_track": "Scarica Brano",
+  "tracks_in_queue": "{tracks} brani in coda",
+  "clear_all": "Pulisci Tutto",
+  "show_hide_ui_on_hover": "Mostra/Nascondi UI al passaggio del mouse",
+  "always_on_top": "Sempre in primo piano",
+  "exit_mini_player": "Esci da Mini player",
+  "download_location": "Cartella Download",
+  "account": "Profilo",
+  "login_with_spotify": "Entra con il tuo profilo Spotify",
+  "connect_with_spotify": "Connettiti con Spotify",
+  "logout": "Esci",
+  "logout_of_this_account": "Esci da questo profilo",
+  "language_region": "Lingua & Paese",
+  "language": "Lingua",
+  "system_default": "Default di Sistema",
+  "market_place_region": "Paese del Marketplace",
+  "recommendation_country": "Recommendation Country",
+  "appearance": "Aspetto",
+  "layout_mode": "Layout Mode",
+  "override_layout_settings": "Override responsive layout mode settings",
+  "adaptive": "Adattabile",
+  "compact": "Compatto",
+  "extended": "Esteso",
+  "theme": "Tema",
+  "dark": "Scuro",
+  "light": "Chiaro",
+  "system": "Sistema",
+  "accent_color": "Accent Color",
+  "sync_album_color": "Sync album color",
+  "sync_album_color_description": "Usa il colore principale della copertina dell'album come accent color",
+  "playback": "Playback",
+  "audio_quality": "Qualità Audio",
+  "high": "Alta",
+  "low": "Bassa",
+  "pre_download_play": "Pre-download e riproduci",
+  "pre_download_play_description": "Instead of streaming audio, download bytes and play instead (Recommended for higher bandwidth users)",
+  "skip_non_music": "Salta le parti 'non musicali' (SponsorBlock)",
+  "blacklist_description": "Brani nella Blacklist e artisti",
+  "wait_for_download_to_finish": "Aspetta che il download in corso finisca",
+  "download_lyrics": "Scarica i testi insieme ai brani",
+  "desktop": "Desktop",
+  "close_behavior": "Close Behavior",
+  "close": "Chiudi",
+  "minimize_to_tray": "Minimize to tray",
+  "show_tray_icon": "Show System tray icon",
```

Comment:
Other software translate "Tray Icon" as "Icona della Barra" or "Icona del Vassoio".
This can be "Mostra Icona nella Barra" or "Mostra Icona nel Vassoio"

Reply:
I chose "Mostra Icona nella Barra", I think it sounds better (I don't like Vassoi 🍽️).

Reply:
Fixed in 940ff32e0144605b4641bb6c538f5870cd7bc011